sktime/transformations/series/summarize.py

```diff
@@ -750,3 +750,140 @@ def get_test_params(cls, parameter_set="default"):
         params4 = {"summary_function": None, "quantiles": (0.1, 0.2, 0.25)}
 
         return [params1, params2, params3, params4]
+
+
+class SplitterSummarizer(BaseTransformer):
+    """Create summary values of a time series' splits.
+
+    A series-to-series transformer that applies a series-to-primitives transformer
+    to each series' train split.
+
+    It splits the series using a specified splitter and transforms each resulting
+    split using the provided transformer.
+
+    This transformer aims to provide a summarization of the series based on the
+    transformer and splitter given.
+
+    Attributes
+    ----------
+    transformer : series-to-primitives transformer used to convert series
+        to primitives.
+
+    splitter : splitter used to divide the series.
+
+    Methods
+    -------
+    transform(X) : Transforms the series according to the specified
+        series-to-primitives transformer and splitter.
+
+    See Also
+    --------
+    SummaryTransformer: Calculates summary value of a time series.
+
+    Notes
+    -----
+    Empty
+
+    Examples
+    --------
+    >>> from sktime.transformations.series.summarize import SplitterSummarizer
+    >>> from sktime.transformations.series.summarize import SummaryTransformer
+    >>> from sktime.forecasting.model_selection import ExpandingWindowSplitter
+    >>> from sktime.datasets import load_airline
+    >>> y = load_airline()
+    >>> transformer = SplitterSummarizer(
+    >>>     transformer=SummaryTransformer(),
+    >>>     splitter=ExpandingWindowSplitter())
+    >>> y_splitsummarized = transformer.fit_transform(y)
+    """
+
+    _tags = {
+        "scitype:transform-input": "Series",
+        # what is the scitype of X: Series, or Panel
+        "scitype:transform-output": "Series",
+        # what scitype is returned: Primitives, Series, Panel
+        "scitype:instancewise": True,  # is this an instance-wise transform?
+        "X_inner_mtype": ["pd.DataFrame", "pd.Series"],
+        # which mtypes do _fit/_predict support for X?
+        "y_inner_mtype": "None",  # which mtypes do _fit/_predict support for X?
+        "fit_is_empty": True,
+    }
+
+    def __init__(self, transformer, splitter, index="last"):
```

Comment:
maybe `ExpandingWindowSplitter` is the most sensible default for `splitter`?

If we want this default, then the `__init__` default should be `None`, and the default is written to `self._splitter`.

Reply:
Made changes regarding splitter's default object with  `ExpandingWindowSplitter` . I am thinking maybe `SlidingWindowSplitter(window_length=10, step_length=10, fh=1, start_with_window=True)` would be more appropriate for a default splitter here ?

Reply:
Hm, let's think carefully in terms of user expectations. A good default has no "surprising" choices, i.e., it agrees with the user guess if you don't tell them what the choice is.

* I agree that sliding window is more in line with user expectation than expanding.
* however, sliding has the problem of having to specify window and step length.
* I disagree with `step_length=10` because it is an arbitrary choice, and default user expectation imo is `step_length=1`.
* `window_length` is also arbitrary, but consistent with the default of `SlidingWindowSplitter`. Perhaps not overriding the default is the best choice here.

Reply:
I think you did not change this?